rasa/nlu/extractors/extractor.py

```diff
@@ -254,6 +268,40 @@ def _update_confidence_values(
                 confidences[ENTITY_ATTRIBUTE_GROUP][idx],
             )
 
+    @staticmethod
+    def _check_is_single_entity(
+        text: Text,
+        token: Token,
+        last_token_end: int,
+        split_entities_config: Dict[Text, bool],
+        current_entity_tag: Text,
+    ):
+        # Tokens need to be no further than 3 positions apart
+        tokens_within_range = token.start - last_token_end <= 3
```

Comment:
why 3?

Reply:
basically to catch things like `Schönhauser Allee 175, 10119 Berlin` (2 tokens apart) as well as abbreviated addresses like, `22 Powderhall Rd., EH7 4GB` which are 3 tokens apart.

I am happy to take other suggestions, but 3 seemed to be the tightest bound that will work in most cases.

Reply:
make sense, would just add a comment, otherwise this is just a magic number